### PR TITLE
feat: Added support to run local tools with Groq Client

### PR DESCRIPTION
**Pull request by pulphix**
Added Support to run local tools for Groq Client

### CODE REVIEW

File: examples/sample_run_local_tools_groq.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+TOKEN = os.getenv("GROQCLOUD_API_KEY")
+TH_TOKEN = os.getenv("TOOLHOUSE_BEARER_TOKEN")
+
+
+local_tools = [
+    {'type': 'function',
+     'function':
+         {
+             'name': 'hello',
+             'description': 'The user receive a customized hello message from a city and return it to the user',
```
Comment (iamdaniele) [inline examples/sample_run_local_tools_groq.py]: Grammar errors. Should be: “The user receives a customized hello message from a city and returns it to the user.”

File: examples/sample_run_local_tools_groq.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+         }}]
+
+th = Toolhouse(access_token=TH_TOKEN, provider="openai")
+th.set_metadata("id", "fabio")
+th.set_metadata("timezone", 5)
+
+
+@th.register_local_tool("hello")
+def whatever(city: str):
+    """Return Local Time"""
```
Comment (iamdaniele) [inline examples/sample_run_local_tools_groq.py:34]: Change this so it’s not confusing to the user and it describes the function for what it actually does.

**Review by iamdaniele — CHANGES_REQUESTED** [review]
Few typos/grammar errors. Also while you’re at it, find all occurrences of “antropic” and replace them “anthropic”

File: examples/sample_run_local_tools_groq.py (hunk 1 above)
Comment (pulphix) [reply examples/sample_run_local_tools_groq.py]: Fixed

**Pull request merged**